README.md

```diff
@@ -4,30 +4,26 @@
 
 # `ascli` : a Command Line for IBM Aspera products
 
-Version : 1.0.0
+Version : 4.0.0
 
 
 _Laurent/2016-2020_
 
 This gem provides a command line interface to Aspera Applications and a ruby API to Aspera transfers.
 
-Location:
-[https://rubygems.org/gems/aspera](https://rubygems.org/gems/aspera)
+Location (once released):
+[https://rubygems.org/gems/aspera-cli](https://rubygems.org/gems/aspera-cli)
 
 Disclaimers:
 
-* Aspera, FASP are owned by IBM
-* This GEM is not endorsed/supported by IBM/Aspera
-* Use at your risk (not in production environments)
-* This gem is provided as-is, and is not intended to be a complete CLI, or industry-grade product.
-* some features may not be fully validated
-* IBM provides an officially supported Aspera CLI: [http://downloads.asperasoft.com/en/downloads/62](http://downloads.asperasoft.com/en/downloads/62) .
+* This as not yet been officially released so things may change
```

Comment:
minor typo here

Reply:
> minor typo here

Thanks! GH is cool, with a suggestion (from me or you), I just had to click on accept suggestion and bam! it's committed and the PR updated...